chia/full_node/mempool.py

```diff
@@ -403,15 +404,30 @@ def create_bundle_from_mempool_items(
                 continue
             try:
                 cost = uint64(0 if item.npc_result.conds is None else item.npc_result.conds.cost)
-                unique_coin_spends, cost_saving, unique_additions = eligible_coin_spends.get_deduplication_info(
-                    bundle_coin_spends=item.bundle_coin_spends, max_cost=cost
-                )
+                if skipped_items >= 3:
```

Comment:
I think a comment is warranted here, explaining what's going on.
If we've encountered 3 transactions that don't fit in the remaining block size, we want to keep looking for smaller transactions that might fit, but to avoid spending too much time on expensive transactions, we start to take short cuts.